packages/Input/Input.jsx

```diff
@@ -70,6 +74,9 @@ Input.propTypes = {
    *
    * **Deprecation:** As of v1.0.0, passing `Input.Helper` will no longer work.
    *
+   * **Deprecation:** As of v2.0.0, it will no longer exist, as it is not a recommended pattern.
```

Comment:
This appears to be a moving target on our part. We sunset features based on adoption, not version. Let's rephrase this to:
> This is not a recommended pattern and will be removed in a future release.